code/modules/clothing/shoes/magboots.dm

```diff
@@ -91,3 +92,12 @@
 	if(item_flags & NOSLIP)
 		state = "enabled"
 	user << "Its mag-pulse traction system appears to be [state]."
+	
+/obj/item/clothing/shoes/magboots/cheap
+	name = "budget magboots"
+	desc = "Cheap magnetic boots, often used during extravehicular activity to ensure the user remains safely attached to the vehicle. Unlike the high quality variant, these magboots are uncomfortably small, and are not designed to fit non-humanoid feet."
+	magslowdown = 6

```

Comment:
This is way too extreme. Consider 4, maybe 5 at the most.

Reminder that this stuff doesn't need to be right in your face. One notch is generally enough to notice people lagging behind others.